ops/pebble.py

```diff
@@ -67,22 +67,34 @@ def http_open(self, req):
         return self.do_open(_UnixSocketConnection, req, socket_path=self.socket_path)
 
 
-# Matches yyyy-mm-ddTHH:MM:SS.sss[-+]zz(:)zz
+# Matches yyyy-mm-ddTHH:MM:SS.sssZZZ
 _TIMESTAMP_RE = re.compile(
-    r'(\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2})\.(\d+)([-+])(\d{2}):?(\d{2})')
+    r'(\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2})\.(\d+)(.*)')
```

Comment:
If I'm reading this correctly, it is, essentially, skipping the parsing of the timezone in the regex and then moving that into a separate step.

Reply:
That's correct. The `(.*)` collects anything after the seconds+fractional seconds, and the explicit test for `Z` and then the `_TIMEOFFSET_RE` regex handles the time zone offset itself.